docs/installation/sources/sources_linux.rst

```diff
@@ -382,18 +392,18 @@ Compiling Fast DDS-Gen
 
 Once the requirements above are met, compile *Fast DDS-Gen* by following the steps below:
 
+.. note::
+
+    If already installed FastDDS with colcon, you may skip the git clone command; *fastddsgen* can be found under
+    the :code:`src` directory of FastDDS colcon workspace.
```

Comment:
```suggestion
    If Fast DDS has already been installed following :ref:`colcon_installation_linux`, skip cloning Fast DDS-Gen repository, as it can be already be found under the :code:`src` directory within the colcon workspace.
```